bitnami/rabbitmq/templates/secrets.yaml

```
@@ -20,12 +20,17 @@ data:
   rabbitmq-erlang-cookie: {{ include "common.secrets.passwords.manage" (dict "secret" (include "common.names.fullname" .) "key" "rabbitmq-erlang-cookie" "length" 32 "providedValues" (list "auth.erlangCookie") "context" $) }}
   {{ end }}
 {{- end }}
+{{- $extraSecretsPrependReleaseName := .Values.extraSecretsPrependReleaseName }}
 {{- range $key, $value := .Values.extraSecrets }}
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ ternary (printf "%s-%s" (include "common.names.namespace" $) $key) $key $.Values.extraSecretsPrependReleaseName }}
+  {{- if $extraSecretsPrependReleaseName }}
+  name: {{ $.Release.Name }}-{{ $key }}
+  {{- else }}
+  name: {{ $key }}
+  {{- end }}
```

I think this change is simpler and you don't need to define the variable

```suggestion
  name: {{ ternary (printf "%s-%s" $.Release.Name $key) $key $.Values.extraSecretsPrependReleaseName }}
```